From 5e2b9d0dd860813688fc67675f16ea4235c2be91 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Thu, 20 Aug 2015 10:11:25 +0300
Subject: [PATCH 0339/1240] config: added config for spi-net driver (ENC28J60)

- Added config for spi-net driver (ENC28J60)
- Added menu for misc driver in menuconfig
- Added menu for Net driver in menuconfig

Change-Id: I460c2c4b4b60ca3950cfeaaf3fc5fabbd07a735c
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/22982
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Yehuda Yitschak <yehuday@marvell.com>
---
 drivers/misc/Kconfig |  4 ++++
 drivers/net/Kconfig  | 11 +++++++++++
 2 files changed, 15 insertions(+)

diff --git a/drivers/misc/Kconfig b/drivers/misc/Kconfig
index d138724..cbd95ab 100644
--- a/drivers/misc/Kconfig
+++ b/drivers/misc/Kconfig
@@ -1,3 +1,5 @@
+menu "Misc drivers"
+
 config MVEBU_CCU
 	bool "CCU address decoding driver"
 	default n
@@ -46,3 +48,5 @@ config MVEBU_FLC
 	  for Final Level Cache for the Marvell McKinley
 	  Memory Controller driver.
 	  This driver enable FLC and open the FLC windows.
+
+endmenu
diff --git a/drivers/net/Kconfig b/drivers/net/Kconfig
index e69de29..f146802 100644
--- a/drivers/net/Kconfig
+++ b/drivers/net/Kconfig
@@ -0,0 +1,11 @@
+menu "Network drivers"
+
+config ENC28J60
+	bool "ENC28J60 driver"
+	default n
+	help
+	  Choose this option to add support
+	  for eth-spi (enc28j60) which
+	  enable receiving ethernet packets
+	  using spi interface.
+endmenu
-- 
1.9.1

